library/Zend/Code/Reflection/MethodReflection.php

```diff
@@ -460,11 +461,17 @@ protected function isValidFunction($haystack, $position, $functionName = null)
         return $isValid;
     }
 
+    /**
+     * @return string
+     */
     public function toString()
     {
         return parent::__toString();
     }
 
+    /**
+     * @return string
+     */
     public function __toString()
```

Comment:
If this method only calls `parent::__toString()`, it has no use and can be removed.